frontend/src/components/CreateResumeForm.jsx

```
@@ -0,0 +1,55 @@
+        }
+        setError("")
+        try{
+            const response=await axiosInstance.post(API_PATHS.RESUME.CREATE,{
+                title,
+            })
+            if(response.data?._id){
+             navigate(`/resume/${response.data?._id}`)
+            }
+        }
+        catch(error){
+            if(error.response && error.response.data.message){
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)